### PR TITLE
Add VictoryHistogram

### PR DESCRIPTION
**Pull request by NgoKnows**
This PR adds the `VictoryHistogram` component. It also adds JS/TS demos of VictoryHistogram and updates relevant packages.

### CODE REVIEW

File: demo/js/package.json [hunk 1]
```diff
@@ -9,34 +9,36 @@
     "react-native-gesture-handler": "^1.3.0",
     "react-native-svg": "9.13.6",
     "react-navigation": "^3.11.1",
+    "victory-histogram": "^34.3.0",
```
Comment (boygirl) [inline demo/js/package.json:12]: Please remove before merging

File: demo/ts/package.json [hunk 2]
```diff
@@ -14,34 +14,36 @@
     "react-native-gesture-handler": "^1.5.6",
     "react-native-svg": "^11.0.1",
     "react-navigation": "3.11.1",
+    "victory-histogram": "^34.3.0",
```
Comment (boygirl) [inline demo/ts/package.json:17]: ditto

**Comment by NgoKnows** [comment]
@boygirl, I addressed your comments, is this good to merge once tests pass?

**Comment by boygirl** [comment]
Yep! Tested locally, and everything looks great.

**Pull request merged**